geopandas/geodataframe.py

```diff
@@ -369,7 +409,23 @@ def from_file(cls, filename, **kwargs):
 
         Examples
         --------
-        >>> df = geopandas.GeoDataFrame.from_file('nybb.shp')
+
+        >>> path = geopandas.datasets.get_path('nybb')
+        >>> gdf = geopandas.GeoDataFrame.from_file(path)
```

Comment:
I am personally not a fan of this alias of `read_file`, so I would maybe even show `geopandas.read_file` in the example here as a stronger recommendation of it (although in which case we should actually also more clearly mention it in the "extended summary" above)